app/models/vim_performance_operating_range.rb

```diff
@@ -6,22 +6,35 @@ class VimPerformanceOperatingRange < ApplicationRecord
 
   DEFAULT_STD_DEV_MULT = 1
 
+  def recalculate_values
+    self.values = Metric::LongTermAverages.get_averages_over_time_period(
+      resource,
+      :avg_days    => days,
+      :ext_options => {:time_profile => time_profile}
+    )
+  end
+
   def values_to_metrics(options = {})
-    options[:std_dev_mult] ||= DEFAULT_STD_DEV_MULT
```

Comment:
Did a look into this prior to this PR, and I believe there is never a situation where `value_to_metrics` is called where an options hash is passed in.  Was this a feature that never came to be?

I didn't even bother doing this when dealing with https://github.com/ManageIQ/manageiq/pull/12955 because of this, but maybe I am missing something.